### ISSUE TITLE
omtose-darker and omtose-softer theme

### ISSUE THREAD

**Issue by Cifer-Y** [syl20bnr/spacemacs#6657]
When I use SPC-T-h to choose omtose-darker/softer is working well.

Then I wirte `omtose-darker` to .spacemacs in `dotspacemacs-themes` and reboot spacemacs.

And the warning is:

```
Warning (spacemacs): An error occurred while retrieving the theme, using default theme. (error: (error Package ‘omtose-darker-theme-’ is unavailable))
```

That is very confused, how can I fix that?

**Comment by TheBB** [comment]
We don't have support for omtose. Please make a PR changing the file core/core-themes-support.el. The other themes in that file should indicate what you need to do.

**Comment by Cifer-Y** [comment]
Ah, my bad, I found other's pr about that.
https://github.com/syl20bnr/spacemacs/pull/6560

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
